runtime/oti/j9port.h

```diff
@@ -965,6 +965,9 @@ typedef struct J9CacheInfoQuery {
 #define J9PORT_MMAP_FLAG_SHARED OMRPORT_MMAP_FLAG_SHARED
 #define J9PORT_MMAP_FLAG_COPYONWRITE OMRPORT_MMAP_FLAG_COPYONWRITE
 #define J9PORT_MMAP_SYNC_WAIT OMRPORT_MMAP_SYNC_WAIT
+#if defined(J9ZOS39064)
+#define J9PORT_MMAP_FLAG_ZOS_64BIT OMRPORT_MMAP_FLAG_ZOS_64BIT
+#endif /* defined(J9ZOS39064) */
```

Comment:
Why shouldn't we just use `OMRPORT_MMAP_FLAG_ZOS_64BIT` instead of creating an alias?

Reply:
We can use `OMRPORT_MMAP_FLAG_ZOS_64BIT`. But I see we are using other J9PORT_MMAP_FLAG_XXX in existing OpenJ9 code, so I add `J9PORT_MMAP_FLAG_ZOS_64BIT`. This way we don't need to check and mix use `J9PORT_` version for some flags and `OMRPORT_` versions for others.